src/io_scene_vrm/importer/vrm_animation_importer.py

```diff
@@ -77,15 +85,17 @@ def build(
             @ Matrix.Diagonal(scale).to_4x4()
         )
 
-        # TODO: 3要素の場合はオイラー角になるか?
-        # TODO: Matrixだったら分解する
```

Comment:
Please don't remove this comment. It's may necessary. If you want to avoid Japanese, please send a translation pull request instead.